src/d3a/setup/strategy_tests/storage_strategy_break_even_hourly.py

```diff
@@ -1,122 +0,0 @@
-"""
```

Comment:
Can you please explain, why you deleted this? Also src/d3a/setup/strategy_tests/storage_strategy_break_even_range.py